Description: Update path for wordlist directory
 This patch updates brutespray.py to use /usr/share/brutespray/.
Author: Stephane Neveu
Forwarded: not-needed
Index: brutespray/brutespray.py
===================================================================
--- brutespray.orig/brutespray.py
+++ brutespray/brutespray.py
@@ -238,7 +238,7 @@ def make_dic_xml():
 
 def brute(service,port,fname,output):
     if args.userlist is None and args.username is None:
-        userlist = 'wordlist/'+service+'/user'
+        userlist = '/usr/share/brutespray/wordlist/'+service+'/user'
         uarg = '-U'
     elif args.userlist:
         userlist = args.userlist
@@ -248,7 +248,7 @@ def brute(service,port,fname,output):
         uarg = '-u'
 
     if args.passlist is None and args.password is None:
-        passlist = 'wordlist/'+service+'/password'
+        passlist = '/usr/share/brutespray/wordlist/'+service+'/password'
         parg = '-P'
     elif args.passlist:
         passlist = args.passlist
